corehq/apps/app_manager/models.py

```diff
@@ -3330,6 +3330,17 @@ def __init__(self, *args, **kwargs):
         if not self.uuid:
             self.uuid = random_hex()
 
+    @classmethod
+    def wrap(cls, doc):
+        from corehq.apps.userreports.util import default_language, localize
+        if isinstance(doc.get('description'), dict):
```

Comment:
Is this for backwards compatibility with apps already using a multi-language description (in which case that'd be good to comment), or for some other purpose?